README.md

```diff
@@ -3,26 +3,34 @@ The Azure App Services extension for VS Code lets you quickly browse, create, ma
 
 ## Features
 
-* Browse sites across all of your Azure subscriptions
-* Browse to the Azure Portal for advanced tasks, such as scaling
-* Create new web apps/deployment slots (Linux with Node.js only)
-* Deploy to your web apps/deployment slots
+  * Browse sites across all of your Azure subscriptions
+  * Browse to the Azure Portal for advanced tasks, such as scaling
+  * Create new web apps/deployment slots (Linux with Node.js only)
+  * Deploy to your web apps/deployment slots
+  * View and edit remote files on your web apps (Preview)
 
-![Deploy to Web App](resources/WebApp_Deploy.png)
-* Start, stop, and restart the web app/deployment slot
-* Swap deployment slots
-* View and edit web app settings
+    ![Deploy to Web App](resources/WebApp_Deploy.png)
```

Comment:
Could you trim the picture a bit? It doesn't need to be that long. :) And it seems to be in the wrong place. Should be under deploy.